Utils/Dataflow/pyDKB/common/__init__.py

```diff
@@ -5,4 +5,5 @@
 from exceptions import *
 import hdfs
 import json_utils as json
+import custom_readline
```

Comment:
(just a suggestion)
Maybe `from custom_readline import custom_readline`, so that in `AbstractProcessorStage.py` (and in any other place) it could be addressed simpty as `from pyDKB.common import custom_readline`?
I think it is more readable way to organize `common` submodule, especially when a sub-submodule contains single homonymous function.

Reply:
I suppose it's a rather good idea. :)